### PR TITLE
ADD analytic_product_category module

### PR DESCRIPTION
**Pull request by angelmoya**
Base Analytic Product Category Categorization
=============================================

This module add Product Category (related with Product itself) on Analytic Entries.

Usage
=====

As Product Category is related with Product, and Analytic Entries has Product,
now you can filter and group by Product Category on Analytic Entries,
on list and reporting view.

### CODE REVIEW

**Comment by angelmoya** [comment]
@rafaelbn

**Comment by rafaelbn** [comment]
OK @angelmoya thanks! cc @chienandalu

File: analytic_product_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+As Product Category is related with Product, and Analytic Entries has Product,
+now you can filter and group by Product Category on Analytic Entries,
+on list and reporting view.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/87/11.0
+
+.. repo_id is available in https://github.com/OCA/account-analytic
```
Comment (chienandalu) [inline analytic_product_category/README.rst]: I think this can be dropped

**Review by chienandalu — APPROVED** [review]
:+1: Thanks!

**Review by oihane — APPROVED** [review]
LGTM

**Comment by angelmoya** [comment]
@pedrobaeza can you check this

File: analytic_product_category/models/analytic.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class AnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    product_category_id = fields.Many2one(
+        comodel_name='product.category',
+        string='Product Category',
+        related='product_id.categ_id',
+        store=True)
```
Comment (pedrobaeza) [inline analytic_product_category/models/analytic.py]: This should be `readonly=True` for avoiding side effects.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: analytic_product_category/models/analytic.py (hunk 2 above)
Comment (angelmoya) [reply analytic_product_category/models/analytic.py]: Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**